src/charm.py

```diff
@@ -270,6 +270,22 @@ def _update(self, event):
 
         self.unit.status = ActiveStatus()
 
+    def _setup_container(self, container: Container):
```

Comment:
useless question: why was this adopted by the class, while the others no? :)
it doesn't seem like it is using self no?

Reply:
You're right, updated. Thanks!